---
permalink: /
title: "ðŸ‘‹ðŸ¼ Hello, I'm Phuong"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am a microbiologist focused on translating omics data into biological insights.

I am currently working as a Research Associate in [Professor Marc Ouelletteâ€™s lab](https://www.crchudequebec.ulaval.ca/en/researcher/marc-ouellette/) at the CHU de QuÃ©bec â€“ UniversitÃ© Laval Research Center.

I earned a PhD in Process Engineering, specializing in Microbiology, from [AgroParisTech](https://www.agroparistech.fr/en), Paris-Saclay University (France). I then completed my postdoctoral training in my current lab, focusing on antimicrobial resistance in *Streptococcus pneumoniae*. 

Previously, I received an MSc in Molecular and Cellular Biology at [Pierre and Marie Curie University](https://www.sorbonne-universite.fr/en) (France), and a BPharm at [Hanoi University of Pharmacy](https://www.hup.edu.vn/en) (Vietnam).

# ðŸ”¬ Selected experiences

My omics journey began during my MSc internship in the GMPA (now [SayFood](https://eng-umr-sayfood.versailles-saclay.hub.inrae.fr/)) laboratory, where I conducted genome sequencing, assembly, and annotation of *Brevibacterium*, a bacterium widely used in surface-ripened cheeses for producing aroma compounds and red-orange pigments. My work on *Brevibacterium* continued during my [PhD](https://theses.fr/2018SACLA035), where I integrated genomic, transcriptomic, and metabolomic analyses to uncover its adaptation mechanisms to the cheese environment.

My postdoctoral training in [Professor Marc Ouelletteâ€™s lab](https://www.crchudequebec.ulaval.ca/en/researcher/marc-ouellette/) allowed me to strengthen my bioinformatics skills by implementing genomic analysis pipelines in a high-performance computing (HPC) environment. My work on [trimethoprim resistance in *Streptococcus pneumoniae*](https://journals.asm.org/doi/10.1128/mbio.01360-24) combined genome-wide association study (GWAS), resistance reconstruction, and machine learning to provide a proof of concept for integrating *in silico* analysis, wet-lab validation and diagnostic tool development to explore antimicrobial resistance (AMR) across various bacteria-antibiotic combinations.

My current research primarily focuses on deciphering the molecular mechanisms of AMR (including multidrug resistance) and adaptation in *S. pneumoniae*. I have been developing, maintaining, and optimizing omics analysis pipelines for my research group and collaborators, targeting various bacteria (e.g., *S. pneumoniae*, *Escherichia coli*, *Salmonella enterica*) and protozoan parasites (*Leishmania*).

# ðŸ“œ Favorite quotes

> *All is data.*

> *Art is me, science is us.* (Claude Bernard)

ðŸ’¬ If my expertise aligns with your program or research interests, I would be happy to discuss potential collaborations. Please feel free to [contact me](https://phng3001.github.io/contact/).