---
title: "Comparative genomic analysis of <i>Brevibacterium</i> strains: insights into key genetic determinants involved in adaptation to the cheese habitat"
collection: publications
category: manuscripts
permalink: /publication/2017-12-07_BMC-Genomics
excerpt: 'This study compared the gene repertoire of 23 <i>Brevibacterium</i> strains to elucidate genetic mechanisms involved in their ability to grow on the cheese surface.'
date: 2017-12-07
venue: 'BMC Genomics'
paperurl: 'https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4322-1'
citation: 'Pham, N. P., Layec, S., Dugat-Bony, E., Vidal, M., Irlinger, F., & Monnet, C. (2017). &quot;Comparative genomic analysis of Brevibacterium strains: insights into key genetic determinants involved in adaptation to the cheese habitat.&quot; <i>BMC genomics</i>, 18(1), 955.'
---

# Background
*Brevibacterium* strains are widely used for the manufacturing of surface-ripened cheeses, contributing to the breakdown of lipids and proteins and producing volatile sulfur compounds and red-orange pigments. The objective of the present study was to perform comparative genomic analyses in order to better understand the mechanisms involved in their ability to grow on the cheese surface and the differences between the strains.

# Results
The genomes of 23 *Brevibacterium* strains, including twelve strains isolated from cheeses, were compared for their gene repertoire involved in salt tolerance, iron acquisition, bacteriocin production and the ability to use the energy compounds present in cheeses. All or almost all the genomes encode the enzymes involved in ethanol, acetate, lactate, 4-aminobutyrate and glycerol catabolism, and in the synthesis of the osmoprotectants ectoine, glycine-betaine and trehalose. Most of the genomes contain two contiguous genes encoding extracellular proteases, one of which was previously characterized for its activity on caseins. Genes encoding a secreted triacylglycerol lipase or involved in the catabolism of galactose and D-galactonate or in the synthesis of a hydroxamate-type siderophore are present in part of the genomes. Numerous Fe3+/siderophore ABC transport components are present, part of them resulting from horizontal gene transfers. Two cheese-associated strains have also acquired catecholate-type siderophore biosynthesis gene clusters by horizontal gene transfer. Predicted bacteriocin biosynthesis genes are present in most of the strains, and one of the corresponding gene clusters is located in a probable conjugative transposon that was only found in cheese-associated strains.

# Conclusions
*Brevibacterium* strains show differences in their gene repertoire potentially involved in the ability to grow on the cheese surface. Part of these differences can be explained by different phylogenetic positions or by horizontal gene transfer events. Some of the distinguishing features concern biotic interactions with other strains such as the secretion of proteases and triacylglycerol lipases, and competition for iron or bacteriocin production. In the future, it would be interesting to take the properties deduced from genomic analyses into account in order to improve the screening and selection of *Brevibacterium* strains, and their association with other ripening culture components.