---
title: Holistic understanding of trimethoprim resistance in <i>Streptococcus pneumoniae</i> using an integrative approach of genome-wide association study, resistance reconstruction, and machine learning
collection: publications
category: manuscripts
permalink: /publication/2024-09-11_mBio
excerpt: This study integrated genome-wide association study, functional validation, and machine learning to investigate trimethoprim (TMP) resistance in <i>S. pneumoniae</i>. Additive mutations in <i>folA</i> and <i>sulA</i> were identified as key drivers of TMP resistance, enabling accurate digital MIC prediction and offering a scalable framework for studying AMR in other pathogens.
date: 2024-09-11
venue: mBio
paperurl: 'https://journals.asm.org/doi/10.1128/mbio.01360-24'
citation: Pham, N.P., Gingras, H., Godin, C., Feng, J., Groppi, A., Nikolski, M., Leprohon, P. and Ouellette, M., (2024). &quot;Holistic understanding of trimethoprim resistance in <i>Streptococcus pneumoniae</i> using an integrative approach of genome-wide association study, resistance reconstruction, and machine learning&quot;. <i>MBio</i>, 15(9), pp.e01360-24.
---

Antimicrobial resistance (AMR) is a public health threat worldwide. Next-generation sequencing (NGS) has opened unprecedented opportunities to accelerate AMR mechanism discovery and diagnostics. Here, we present an integrative approach to investigate trimethoprim (TMP) resistance in the key pathogen *Streptococcus pneumoniae*. We explored a collection of 662 *S. pneumoniae* genomes by conducting a genome-wide association study (GWAS), followed by functional validation using resistance reconstruction experiments, combined with machine learning (ML) approaches to predict TMP minimum inhibitory concentration (MIC). Our study showed that multiple additive mutations in the *folA* and *sulA* loci are responsible for TMP non-susceptibility in *S. pneumoniae* and can be used as key features to build ML models for digital MIC prediction, reaching an average accuracy within Â±1 twofold dilution factor of 86.3%. Our roadmap of *in silico* analysis-wet-lab validation-diagnostic tool building could be adapted to explore AMR in other combinations of bacteria-antibiotic.